variables.tf

```diff
@@ -248,3 +248,14 @@ variable "queues" {
   }))
   default = []
 }
+
+variable "queue_properties_logging" {
```

Comment:
What about having a default value? I think this variable should not be mandatory

Reply:
I think It already have a default value

<img width="438" alt="Screenshot 2022-10-11 at 10 40 59" src="https://user-images.githubusercontent.com/22898667/195041962-df85863a-fa0a-4717-b70c-59577970c74a.png">

Reply:
```suggestion
variable "queue_properties_logging" {
default = {}
```

Reply:
Good catch, I didn't attention to it 😅

Reply:
It's done!